src/Components/Capture.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Capture.h"
+#include "Render.h"
+#include "Encoder.h"
+#include "Decoder.h"
+
+#include <iostream>
+using namespace std;
```

Comment:
Please do not use "using namespace std". For .cpp  file, you can have "using std::XXX" instead.